cloud/releases/cloud-release-notes.md

```diff
@@ -3,6 +3,27 @@ id: cloud-release-notes
 title: Centreon Cloud release notes
 ---
 
+## March 29, 2024
+
+### Custom check command definition
+
+You now have the ability to define your own check commands, allowing for customized monitoring configurations.
+To maintain system integrity, this feature is secured by a whitelist mechanism (this requires a poller update), ensuring only authorized commands are executed.
+
+### Dashboards Beta
+
+As usual, lots of improvements to the dashboard feature with this release:
+- Navigation in dashboards has been enhanced with a new search feature, a listing presentation and a better responsiveness.
```

Comment:
- It is now possible to create dashboards playlists. Selected dashboards will be displayed sequentially without any interaction allowing to broadcast them on TV set on public spaces.